utils/temperusb/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2007-2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=temperusb
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+PKG_CONFIG_DEPENDS:=libusb-compat
```

Comment:
seriously?  you've got new software in 2018 that still uses libusb 0.1?

Reply:
Yes. I am reusing already existing source code. Migrating it to a current version of libusb could be a challenge for me in the coming weeks, but I am not sure I am knowledgeable enough.

All in all, I will correct the trivial errors in this pull request. In any case this is a good exercise. 
Regarding the portage to a current libusb, what is your guidance? What is the right trade-off between clean and current code base, and supported functionalities?